app/presenters/travel_advice_presenter.rb

```diff
@@ -45,6 +45,7 @@ def last_reviewed_or_updated_at
   # has a latest update label, so we can strip this out.
   # Avoids: "Latest update: Latest update - …"
   def latest_update
+    return nil unless change_description.present?
```

Comment:
nil is redundant here: 
`return unless change_description.present?` does the same thing